search.py

```diff
@@ -0,0 +1,23 @@
+# Name of this file: search.py
+# Location: at the root of the project
+# How to execute: python3 -m search.py query
+
+import sys
+import louis.db.api as api
+import louis.db as db
+from microbench import MicroBench
+import pandas as pd
+
+basic_bench = MicroBench()
+
+# Execute the SQL search function 
+@basic_bench
+def search(cursor, query):
+    return api.search_from_text_query(cursor, query)
+
+if __name__ == '__main__':
+    connection = db.connect_db()
+    with db.cursor(connection) as cursor:
+        results = search(cursor, ' '.join(sys.argv[1:]))
+        print(results)
+    print(pd.read_json(basic_bench.outfile.getvalue(), lines=True))
```

Comment:
how will you aggregate multiple runs benchmarks to perform analysis?

expecting a plot comparing before and after your improvements

Reply:
I've found the right option to the benchmarking, this allow the results to be put in a json file that could be used later for analysis purposes.